src/test/scala/org/infinispan/spark/suites/StreamingFailOverSuite.scala

```diff
@@ -32,7 +32,7 @@ class StreamingFailOverSuite extends FunSuite with SparkStream with MultipleServ
 
    protected def getProperties = {
       val props = new Properties()
-      props.put("infinispan.client.hotrod.server_list", s"localhost:$getServerPort")
+      props.put("infinispan.client.hotrod.server_list", s"${Cluster.getServerList}")
```

Comment:
> I thought the test was about having [server0,server1,server2] and make sure you could write to Infinispan even when some of the servers were down

the test was meant to test if connector is able to write to ISPN when some server goes down **and** also if the connector is able to use HR topology information to resume operation when original ISPN server goes down (i.e. it was intentional to provide only one server - when all servers in cluster are provided, test (or some of them, don't remember exactly) are passing without any fixes)